e2e/support/helpers/e2e-collection-helpers.ts

```diff
@@ -7,18 +7,6 @@ import {
 } from "e2e/support/helpers";
 import type { CollectionId } from "metabase-types/api";
 
-/**
- * Clicks the "+" icon on the collection page and selects one of the menu options
- *
- * @deprecated Use newButton helper
- */
-export function openNewCollectionItemFlowFor(
-  type: "question" | "dashboard" | "collection",
-) {
-  cy.findByText("New").click();
-  popover().findByText(new RegExp(type, "i")).click();
-}
-
```

Comment:
This comment was confusing (along with the naming of the function).
It says clicks on the + icon _on the collection page_.

Even the name suggests that this was a button that exists on the EMPTY collection page. But the fact that it was searching for the string "New" tells me that it referred to the top app bar "New" button.

Replaced it with the `newButton` helper as the deprecation warning suggests.